core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2560,41 +2562,6 @@ class ReplicaManagerTest {
     }
   }
 
-  @ParameterizedTest
-  @EnumSource(value = classOf[Errors], names = Array("NOT_COORDINATOR", "CONCURRENT_TRANSACTIONS", "COORDINATOR_LOAD_IN_PROGRESS", "COORDINATOR_NOT_AVAILABLE"))
-  def testMaybeVerificationErrorConversions(error: Errors): Unit = {
```

Comment:
Don't we need to keep this one as we still have those conversion but in a different place now?

Reply:
We have the test above in this file.

Reply:
We also have one for the GroupCoordinator.